src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentParty.kt

```diff
@@ -29,19 +29,19 @@ class IncidentPartyId(
 open class IncidentParty(
 
   @EmbeddedId
-  val id: IncidentPartyId,
+  open val id: IncidentPartyId,
```

Comment:
needed to prevent startup exception

Reply:
I think this needs `@OpenEntity`  annotation possibly - weird I'm not getting the error

Reply:
I get it on startup when running int tests - plus Mike Willis had added open to his class vars for the same pattern